windows/setup.yml

```diff
@@ -3,8 +3,21 @@ user_message: |
   ''
 
 controller_components:
+  - organizations
+  - users
   - projects
   - job_templates
+  - roles
+
+controller_organizations:
+- name: "Helpdesk Org"
```

Comment:
Why do we need a new org? Wouldn't helpdesk just be a different team perhaps in the "windows" org?

Reply:
Was going by the idea that typically helpdesk is a tier 1 support that lives in its own org. They do level 1 triage for all types of issues and typically route the incident to the appropriate SME team (Windows/Network/DB etc).
I'm good with this being a team, long as we can apply the execute only permissions for the job template and ensure that they don't have visibility to the inventory.